src/main/resources/queue_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS `{schedule_table_name}` (
```

Comment:
It is possible to concat two scripts in one file Mysql2o executes two queries in this case